.github/workflows/release.yml

```diff
@@ -0,0 +1,38 @@
+name: Release
+# https://github.com/bent10/monorepo-starter/blob/main/docs/recipes/release-automation.md
+
+on:
+  push:
+    branches:
+      - main
+      - next
+      - next-major
+      - beta
+      - alpha
+      - "*.x"
+
+jobs:
+  release:
+    name: Releasing
+    runs-on: ubuntu-latest
+    if: "!contains(github.event.head_commit.message, '[skip ci]')"
```

Comment:
We don't need it anymore: https://github.blog/changelog/2021-02-08-github-actions-skip-pull-request-and-push-workflows-with-skip-ci/
```suggestion
```